easybuild/easyconfigs/s/Salmon/Salmon-0.8.2_CMAKE_CXX_FLAGS.patch

```diff
@@ -0,0 +1,12 @@
+* don't hardcode $CMAKE_CXX_FLAGS
```

Comment:
@boegel Currently this doesn't fix the linking with Jellyfish. I also saw that it builds it's own BWA. I will see if I can solve this.